### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by klipp86** [beardedspice/beardedspice#156]
Guys! I love BeardedSpice! I use my headphones a lot at work. Can you somehow integrate the Apple EarPod play/pause button? (The middle button on the most recent Apple headphones)? Would LOVE IT. Thanks!

**Comment by madeddie** [comment]
This project has code to catch inline button presses for headphones: https://github.com/Daij-Djan/DDHidLib
We might be able to use some of the logic.

**Comment by Stillness-2** [comment]
@klipp86 Can you check this build: https://www.dropbox.com/s/ubj1dzvqkwtneud/BeardedSpice.zip?dl=1 ?

**Comment by klipp86** [comment]
Hey guys,

This works!!! Wow, great job. I'll let you know if I have any issues with
it, but this works like a charm with Pandora so far. I don't use many other
services but it works smoothly with Pandora.

Thanks so much for including me in the testing, happy to help. (I had
manually adjust the preferences when I first installed 1.2, since the
"apple remote" option was unchecked.)

I would suggest maybe calling it something else besides "apple remote"
because it makes me think you mean the little hand-held remote that came
with older iMacs and MacBooks. Not sure what it's technically called, but
maybe even dumb it down and spell it out:

"Use 'Play-Pause' button on Apple Headphones *to *control playback."

Also, what does the "A" mean next to different media controllers in the
Prefs window?

Thanks!


Kevin Lipp

**Comment by Stillness-2** [comment]
"A" - this is "Automatic". Read this: https://github.com/beardedspice/beardedspice#smart-mode
About apple remote :) Apple Remote - this is "Apple Remote", and preferences item should turn on control of playback using Apple Remote. :)
Playback control using buttons of headphones must work by default.

**Comment by klipp86** [comment]
Not sure I understand the second half of your email.

However, I just found something. I paused my music on Pandora with the
headphone button and walked away for 20 minutes or so. I cam back and I
clicked Un-Pause with Pandora in a background tab (on my headphones). It
resumed playback on Pandora, but also launched iTunes for some reason.


Kevin Lipp

**Comment by klipp86** [comment]
And now it's launching iTunes overtime I click the headphones. Weird! It
was working perfectly upon initial install. Not sure what changed.


Kevin Lipp
